src/events.zig

```diff
@@ -390,6 +395,40 @@ pub const User = struct {
     data2: ?*anyopaque = null,
 };
 
+/// Keyboard button event structure (event.key.*)
+/// ## Remarks
+/// The key is the base SDL_Keycode generated by pressing the scancode using the current keyboard
+/// layout, applying any options specified in SDL_HINT_KEYCODE_OPTIONS. You can get the SDL_Keycode
+/// corresponding to the event scancode and modifiers directly from the keyboard layout, bypassing
+/// SDL_HINT_KEYCODE_OPTIONS, by calling SDL_GetKeyFromScancode().
+///
+/// ## Version
+/// This struct is available since SDL 3.2.0.
+///
+/// ## Code Examples
+///    const key = if (ev == .key_down) ev.key_down.key else break :event_handle;
+///    switch (key) {
+///        .a => {
+///            std.debug.print("a\n", .{});
+///        },
+///        else => {
+///            std.debug.print("other\n", .{});
+///        },
+///    }
+pub const Key = struct {
+    type: Type,
+    /// Common event information.
+    common: Common,
```

Comment:
Forgot to mention earlier sorry, but please add documentation for the members from https://wiki.libsdl.org/SDL3/SDL_KeyboardEvent.